### PR TITLE
Price plan description for FairFlat, Business, Enterprise

### PR DESCRIPTION
**Pull request by michelfernandez**
We add 3 new sites for price plan descriptions, one for FairFlat, Business and Enterprise.

### CODE REVIEW

**Comment by fursinger** [comment]
@fursinger todo:

- [ ] Make sure `more-href` and `more-target` are props of the price component and work accordingly.
- [ ] Make pricing structures same height. Use either language specific css pseudo selector to adjust or investigate using a flex layout with align-stretch.

File: site/content/german/enterprise.md [hunk 1]
```diff
@@ -10,14 +10,14 @@ og_image: og/og-skribble-enterprise.jpg
 ---
 {{< intro top=24 bottom=12 width=wide img=enterprise-main img-width=600 img-alt="Ein Mann unterschreibt einen Vertrag im Büro mit seinem Notebook." >}}
 {{% markdown %}}
-Führende Technologie kombiniert 
+Führende Technologie kombiniert
```
Comment (Kirastina) [inline site/content/german/enterprise.md:13]:
```suggestion
Führende Technologie kombiniert mit
```

File: site/content/german/enterprise.md [hunk 2]
```diff
@@ -10,14 +10,14 @@ og_image: og/og-skribble-enterprise.jpg
...
 mit Premium-Service
```
Comment (Kirastina) [inline site/content/german/enterprise.md:14]:
```suggestion
Premium-Service
```

**Review by Kirastina — APPROVED** [review]

File: site/content/french/enterprise.md [hunk 3]
```diff
@@ -11,15 +11,15 @@ og_image: og/og-skribble-enterprise.jpg
 {{< intro top=24 bottom=12 width=wide img=enterprise-main img-width=600 img-alt="Ein Mann unterschreibt einen Vertrag im Büro mit seinem Notebook." >}}
 {{% markdown %}}
 [FR]
```
Comment (Kirastina) [inline site/content/french/enterprise.md:13]:
```suggestion
```

File: site/content/english/business.md [hunk 4]
```diff
@@ -11,15 +11,15 @@ og_image: og/og-skribble-business.jpg
 {{< intro top=24 bottom=12 width=wide img=business-main img-width=600 img-alt="Eine Frau signiert elektronisch auf ihrem Notebook von ihrem Büro aus einen Vertrag." >}}
 {{% markdown %}}
 [EN]
```
Comment (Kirastina) [inline site/content/english/business.md:13]:
```suggestion
```

**Review by Kirastina — APPROVED** [review]

**Comment by fursinger** [comment]
A remark: I always find it a bit confusing, when you can click on the dropdown trigger itself `pricing` and on its subitems `Fair Flat`, `Business`, `Enterprise`. Is it clear enough to the visitor that `pricing` can be clicked on? Maybe nothing to add to the scope now, but we should discuss it - or test it @michelfernandez ;)

**Comment by fursinger** [comment]
My 2nd task can only be done after all the content on the pricing page is finished. @Kirastina pls inform me once that is the case and I'll update the height of the cards to match for each language.

**Comment by Kirastina** [comment]
> My 2nd task can only be done after all the content on the pricing page is finished. @Kirastina pls inform me once that is the case and I'll update the height of the cards to match for each language.

**Comment by Kirastina** [comment]
> > My 2nd task can only be done after all the content on the pricing page is finished. @Kirastina pls inform me once that is the case and I'll update the height of the cards to match for each language.

I'll let you know!

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

**Tested: NOT OK**

I found following bugs:

1. In english and french you see a german text on the "Fair Flat" page

cove_new_pricing_pages_bug_1
cove_new_pricing_pages_bug_2

2. In english you see the text in german on the "Business" page

cove_new_pricing_pages_bug_3

3. The sentence "Bleiben Sie im Ferienhaus" sounds a little bit weird from my point of view. In english it sounds better "Sign electronically from the comfort of your holiday home" I would personally change the sentence in german. I also think that it the french sentence maybe is weird, but I can't speak/read it.

cove_new_pricing_pages_bug_4

4. In german I wouldn't write "Globale Compliance" I think there are some german words that could be used here.

cove_new_pricing_pages_bug_5

5. On mobile in landscape view I see the company logos almost together again. Had this multiple times in the past few page changes.

Screenshot_20201014-125020_Chrome

I would at least fix the first 2 bugs, so I could give a tested:OK label.

File: site/content/french/fair-flat.md [hunk 5]
```diff
@@ -30,7 +30,8 @@ Signez vos documents numériquement de manière juridiquement valable, où que v
 
 [//]: # (--------------------------------------------------------------------------------------------------------------)
 
-{{< logos-container top=12 bottom=24 width=wide title="Die erste Wahl von mehr als 250 Unternehmen mit höchsten Ansprüchen an Datenschutz und Sicherheit:">}}
+{{< logos-container top=12 bottom=24 width=wide title="Le premier choix de plus de 250 entreprises avec les normes les plus élevées en matière de protection des données et de sécurité:
+">}}
```
Comment (Kirastina) [inline site/content/french/fair-flat.md:34]:
```suggestion
```

File: site/content/french/fair-flat.md [hunk 6]
```diff
@@ -30,7 +30,8 @@ Signez vos documents numériquement de manière juridiquement valable, où que v
 
 [//]: # (--------------------------------------------------------------------------------------------------------------)
 
-{{< logos-container top=12 bottom=24 width=wide title="Die erste Wahl von mehr als 250 Unternehmen mit höchsten Ansprüchen an Datenschutz und Sicherheit:">}}
+{{< logos-container top=12 bottom=24 width=wide title="Le premier choix de plus de 250 entreprises avec les normes les plus élevées en matière de protection des données et de sécurité:
```
Comment (Kirastina) [inline site/content/french/fair-flat.md:33]:
```suggestion
{{< logos-container top=12 bottom=24 width=wide title="Le premier choix de plus de 250 entreprises avec les normes les plus élevées en matière de protection des données et de sécurité:">}}
```

**Comment by Kirastina** [comment]
@dkumpf1 Danke fürs super Feedback!

1. Die vergessenen Übersetzungen hab ich eingefügt
2. Die vergessenen Übersetzungen hab ich eingefügt
3. Titel "Bleiben Sie im Ferienhaus": Ist nicht ideal, ist mir bewusst. Wir studieren noch an einer besseren Formulierung rum. Das sollte aber nicht den ganzen PR davon abhalten, live zu gehen. V.a. weil Fair Flat nicht unser Fokus ist (wir machen mit den Business- und den Enterprise-Kunden Geld)
4. Globale Compliance: Ist wirklich nicht besonders Deutsch. Haben uns aber am Wortschatz orientiert, den Enterprise-Kunden so nutzen. Und dort ist "Compliance" ein gängiges Wort.
5. Enger-Logo-Teppich: Gebe ich Michel weiter, für wenn er zurück ist von den Ferien. Für den Moment können wir es so belassen, weil nicht katastrophal.

**Comment by Kirastina** [comment]
We will probably adapt the "Bleiben Sie im Ferienhaus" title later today or tomorrow. But this will be a pure content change, so no need fo further testing by @dkumpf1 after he gave the tested ok.

**Comment by dkumpf1** [comment]
I'm fine with that, as I mentioned in my comment above. Thanks for fixing the bugs quickly @Kirastina

**Pull request merged**